### ISSUE TITLE
[0.20.0] Changing players resets the video.

### ISSUE THREAD

**Issue by kubo6472** [TeamNewPipe/NewPipe#4408]
### Checklist

- [x] I am using the latest version. 
- [x] I checked, but didn't find any duplicates of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

Open a video
Switch to background player

### Actual behaviour

The video starts playing from the beginning

### Expected behavior

The video should continue playing on the timestamp.
This has never happened to me before the unified player, so I'm surprised, that when that's one of the points of having it, it starts breaking stuff.

### Screenshots/Screen recordings

![Screenshot_20201005-152356_NewPipe](https://user-images.githubusercontent.com/26146502/95085390-7304b100-071f-11eb-92ef-d9d1e2e9806f.jpg)

**Comment by opusforlife2** [comment]
Closing until more information arrives.

**Comment by kubo6472** [comment]
It happened only once watching this video: https://youtu.be/bE9Gz4xKgmA 
when I specifically turned the device horizontally, then vertically, 
then clicked on background play. Was not able to replicate it 10/10 
times I tried. I am sure it was not an user action.

**Comment by opusforlife2** [comment]
Make sure to inform us if you do find a way to reproduce it.

**Comment by RickyM7** [comment]
@opusforlife2 This bug occurred with me also in v0.20.1, to make this bug occur you have to minimize the app and come back soon after, the bug only occurs if the video is still loading in the background, if the video loads the bug does not occur, at least not in this video.
Now a bug off topic, if the setting (Settings> Video and audio> Minimize on app swich) is set to "None", if you minimize the app while watching the video, when you return to the app it will close on its own. No error is shown, the app simply closes.

**Comment by opusforlife2** [comment]
@RickyM7 That sounds like a different bug. Please open a new issue.

**Comment by Stypox** [comment]
@kubo6472 could you test if the apk in #4562 fixes the issue?

**Comment by kubo6472** [comment]
That works fine.

**Comment by opusforlife2** [comment]
I thought you only encountered it once and weren't able to reproduce it. Isn't it too soon to know if it was fixed?

**Comment by kubo6472** [comment]
I haven't said that was fixed. I said that I haven't encountered any errors yet.

**Comment by opusforlife2** [comment]
Alright. Make sure to let us know if you encounter it again.

**Comment by RickyM7** [comment]
@opusforlife2 Sorry for the delay. I believe it is no longer necessary for me to open a new issue, after all I tested the apk in #4562 and the two bugs that I reported were resolved in that apk.
